pkg/api/v1beta3/dynakube/telemetryservice/props.go

```diff
@@ -35,6 +37,10 @@ func (spec *Spec) GetProtocols() []Protocol {
 	return protocols
 }
 
+func (ts *TelemetryService) GetName(dynakubeName string) string {
+	return dynakubeName + nameSuffix
+}
```

Comment:
a possible improvement:

you could store the `dk.Name` in the wrapper `TelemtryService`

example:
- [KSPM struct](https://github.com/Dynatrace/dynatrace-operator/blob/f00e14a06a6c72f33d7906ca07e441546802f412/pkg/api/v1beta3/dynakube/kspm/spec.go#L19)
- [How it's set](https://github.com/Dynatrace/dynatrace-operator/blob/f00e14a06a6c72f33d7906ca07e441546802f412/pkg/api/v1beta3/dynakube/kspm_props.go#L11)

Reply:
done